### PR TITLE
Wisconsin county-level data

### PR DESCRIPTION
**Pull request by MattSidor**
County-level scraper for Wisconsin data.

Additional state-level scraper that includes detailed demographic breakdowns will be completed in another pull request.

### CODE REVIEW

**Review by cc7768 — CHANGES_REQUESTED** [review]
Overall this looks great! Thanks for putting together it together.

The one change I explicitly added was including an import in the `can_tools/scrapers/official/__init__.py` file (I added this to get it working while I reviewed the PR).

I think you said that you were going to do the demographic cases/deaths in a separate PR so I will keep an eye out for that.

File: can_tools/scrapers/official/WI/wi_state.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                category="unspecified_tests_positive",
+                measurement="new",
+                unit="unique_people",
+            ),
+            "test_new": CMU(
+                category="unspecified_tests_total",
+                measurement="new",
+                unit="unique_people",
+            ),
+        }
```
[the remark this answers is not in the document]
Comment (MattSidor) [reply can_tools/scrapers/official/WI/wi_state.py:60]: Yes, you're right -- "Antigen and antibody test results are not included in this file."  Thank you.

File: can_tools/scrapers/official/WI/wi_state.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        out = (
+            df.melt(id_vars=["location"], value_vars=crename.keys())
+            .assign(
+                dt=self._retrieve_dt("US/Eastern"), vintage=self._retrieve_vintage()
```
[the remark this answers is not in the document]
Comment (MattSidor) [reply can_tools/scrapers/official/WI/wi_state.py:64]: Thanks. For states spread across multiple time zones -- e.g. counties in the Florida panhandle; some counties in the northwestern/southwestern corners of Indiana -- should we just use the state's "main" timezone for all counties?

[the remark this answers is not in the document]
Comment (cc7768) [reply can_tools/scrapers/official/WI/wi_state.py]: That's a good question -- I'll propose that we use the timezone of the state capital (so for FL that would be Jacksonville which is Eastern).

I'm open to other thoughts on this though

Ping @sglyon

[the remark this answers is not in the document]
Comment (sglyon) [reply can_tools/scrapers/official/WI/wi_state.py]: That sounds great. Good question and answer

**Comment by sglyon** [comment]
Hey green light from CI :tada:

**Review by cc7768 — APPROVED** [review]

**Pull request merged**